src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1367,6 +1367,25 @@ private function addHttpClientSection(ArrayNodeDefinition $rootNode)
                     ->info('HTTP Client configuration')
                     ->{!class_exists(FullStack::class) && class_exists(HttpClient::class) ? 'canBeDisabled' : 'canBeEnabled'}()
                     ->fixXmlConfig('scoped_client')
+                    ->beforeNormalization()
+                        ->always(function ($config) {
+                            if (empty($config['scoped_clients']) || !\is_array($config['default_options']['retry_failed'] ?? null)) {
```

Comment:
`empty($config['scoped_clients'])` -> `!$config['scoped_clients']`

Reply:
key `scoped_clients` may not exist in `$config`. 

I can use `!isset($config['scoped_clients']) || !$config['scoped_clients']` but it's less readable IMHO